content/v2.0/security/https_setup.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Enabling HTTPS with InfluxDB
+description: >
+  Enable HTTPS and Transport Security Layer (TLS) secure communication between clients and your InfluxDB servers.
+weight: 201
+menu:
+  v2_0:
+    name: Enabling HTTPS
+    parent: Security & authorization
+v2.0/tags: [security, authentication, tls, https]
+---
+
+Enabling HTTPS encrypts the communication between clients and the InfluxDB server.
+When configured with a signed certificate, HTTPS can also verify the authenticity of the InfluxDB server to connecting clients.
+
+{{% warn %}}
+InfluxData [strongly recommends](/influxdb/v1.7/administration/security/) enabling HTTPS, especially if you plan on sending requests to InfluxDB over a network.
+{{% /warn %}}
+
+## Requirements
+
+To enable HTTPS with InfluxDB, you'll need an existing or new InfluxDB instance
+and a Transport Layer Security (TLS) certificate (also known as a Secured Sockets Layer (SSL) certificate).
+
+InfluxDB supports three types of TLS certificates:
+
+* **Single domain certificates signed by a Certificate Authority**
+
+    Single domain certificates provide cryptographic security to HTTPS requests and allow clients to verify the identity of the InfluxDB server.
+    With this certificate option, every InfluxDB instance requires a unique single domain certificate.
+
+* **Wildcard certificates signed by a Certificate Authority**
+
+    Wildcard certificates provide cryptographic security to HTTPS requests and allow clients to verify the identity of the InfluxDB server.
+    Wildcard certificates can be used across multiple InfluxDB instances on different servers.
+
+* **Self-signed certificates**
+
+    Self-signed certificates are _not_ signed by a Certificate Authority (CA).
```

Comment:
I think you should define and abbreviate "Certificate Authority" sooner than this. If you do that, just use "CA" here.

Reply:
OK, this is where x509 gets a little murky :joy: 

Self-signed certificates are/can be signed by a CA (You generate your own), it's just not "trusted"